docs/formatting.md

```diff
@@ -46,6 +46,8 @@ format_multirun(
 )
 ```
 
+File discovery for each language is based on file extension - files with no extension will not be formatted.
```

Comment:
could you add a link to the discussion about fixing this by sniffing the shebang line, just to make it discoverable for anyone who reads this and has the same problem you encountered?

Reply:
Done.